mlir/test/Dialect/MIOpen/Tuning/db_connection.mlir

```diff
@@ -1,3 +1,5 @@
-// RUN: mlir-miopen-driver -p -miopen-lowering -miopen-affix-params -debug %s 2>&1 | FileCheck %s
+// RUN: mlir-miopen-driver -p -miopen-lowering -miopen-affix-params -debug %s 2>&1 | FileCheck %s --check-prefix=CHECK1
+// RUN: mlir-miopen-driver -p -t f16 -miopen-lowering -miopen-affix-params -debug %s 2>&1 | FileCheck %s --check-prefix=CHECK2
 
-// CHECK: Successfully opened connection to PerfDb
+// CHECK1: Successfully opened connection to PerfDb
```

Comment:
Please remove all f16 related `Successfully opened connection to PerfDb`. They are payload independent.

You can change some of the existing test case to use f16 but there's no need to add test coverage for it.